digiroad2-oracle/src/main/resources/db/migration/V227__update_bus_stop_names.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE ENUMERATED_VALUE SET NAME_FI = 'Linja-autopysäkki' WHERE NAME_FI = 'Linja-autojen paikallisliikenne';
+UPDATE ENUMERATED_VALUE SET NAME_FI = 'Kaukoliikenne (vanha tll)' WHERE NAME_FI = 'Linja-autojen kaukoliikenne';
+UPDATE ENUMERATED_VALUE SET NAME_FI = 'Pikavuoro (vanha tll)' WHERE NAME_FI = 'Linja-autojen pikavuoro';
```

Comment:
please add the AND for propertyID.
Otherwise it seems to do the update to all table.

Reply:
Done